Sensor.dcm

```diff
@@ -8,19 +8,19 @@ $ENDCMP
 #
 $CMP AS3935
 D Programmable fully integrated Lightning Sensor IC
-K lightening sensor
+K lightning sensor
 F https://www.embeddedadventures.com/datasheets/AS3935_Datasheet_EN_v2.pdf
 $ENDCMP
 #
 $CMP BME280
-D It is a digital 3 in 1 sensor with humidity, pressure and temperature measurements, LGA-8
-K BME280 pressure humidity temperatiure measurement
+D 3-in-1 humidity, pressure and temperature sensor, I2C and SPI interface, 1.71-3.6V, LGA-8
```

Comment:
Description should be comma separated. I suggest
`3-in-1 sensor, humidity, pressure, temperature, I2C and SPI interface, 1.71-3.6V, LGA-8`